packages/shared/src/lib/core/network/classes/evm-network.class.ts

```diff
@@ -2,12 +2,9 @@ import { IPureEvmNetworkConfiguration } from '../interfaces'
 import { BaseEvmNetwork } from './base-evm-network.class'
 
 export class EvmNetwork extends BaseEvmNetwork {
```

Comment:
We can remove BaseEvmNetwork in favor of EvmNetwork, since we don't have anything ISC L2 EVM specific implemented.

Reply:
will be done in a dedicated PR, as it also touches `Base/Pure EvmConfiguration`